test/mock-results.js

```diff
@@ -15,17 +15,30 @@ module.exports = function(stats, options) {
         testsuite: [
           {
             _attr: {
-              name: "Foo Bar module",
+              name: "Root Suite",
               timestamp: stats.start.toISOString().substr(0,stats.start.toISOString().indexOf('.')),
-              tests: "2",
+              tests: "0",
+              failures: "0",
+              time: "0.0000"
+            }
+          }
+        ]
+      },
+      {
+        testsuite: [
+          {
+            _attr: {
+              name: "Foo Bar",
+              timestamp: stats.start.toISOString().substr(0,stats.start.toISOString().indexOf('.')),
+              tests: "3",
               failures: "2",
               time: "32.1060"
             }
           },
           {
             testcase: {
               _attr: {
-                name: "Foo can weez the juice",
+                name: "Foo Bar can weez the juice",
```

Comment:
The stubbed testcase names weren't accurate. They didn't match the enclosing suites' names.